source/scripting_v3/GTA/World.cs

```diff
@@ -1475,19 +1475,30 @@ public static void ShootBullet(Vector3 sourcePosition, Vector3 targetPosition, P
 		/// <param name="type">The type of explosion.</param>
 		/// <param name="radius">The radius of the explosion.</param>
 		/// <param name="cameraShake">The amount of camera shake to apply to nearby cameras.</param>
+		/// <param name="aubidble">if set to <see langword="true" /> explosion can be heard.</param>
+		/// <param name="invisible">if set to <see langword="true" /> explosion is invisible.</param>
+		/// <param name="noDamage">if set to <see langword="true" /> explosion does not cause damage.</param>
+		public static void AddExplosion(Vector3 position, ExplosionType type, float radius, float cameraShake, bool aubidble = true, bool invisible = false, bool noDamage = false)
```

Comment:
The method signature is being changed and thus this PR is braking the compatibility. Adding optional parameters also changes method signatures.